gpcontrib/arenadata_toolkit/isolation2/expected/track_concurrent.out

```diff
@@ -0,0 +1,57 @@
+-- start_matchsubs
+-- m/ERROR:  Track for database \d+ is being acquired in other transaction/
+-- s/\d+/XXX/g
+-- end_matchsubs
+-- Test concurrent track acquisition.
+1: CREATE EXTENSION IF NOT EXISTS arenadata_toolkit;
+CREATE
+1: SELECT arenadata_toolkit.tracking_register_db();
+ tracking_register_db 
+----------------------
+ t                    
+(1 row)
+1: SELECT arenadata_toolkit.tracking_trigger_initial_snapshot();
+ tracking_trigger_initial_snapshot 
+-----------------------------------
+ t                                 
+(1 row)
+1: BEGIN;
+BEGIN
+1: SELECT tt.segid, count(*) FROM arenadata_toolkit.tables_track tt GROUP BY tt.segid;
+ segid | count 
+-------+-------
+ 1     | 264   
```

Comment:
Can we consider this check future proof? I mean that each time we do some changes in the GPDB core that alters the initial content of the DB, and thus the  count of rows in the `tables_track`, we'll get this test broken. Or we can get into such a situation only if we change the system catalog (meaning we changed major version)?